cmake/thirdparty/get_cccl.cmake

```
@@ -12,11 +12,10 @@
 # the License.
 # =============================================================================
 
-# Use CPM to find or clone thrust
-function(find_and_configure_thrust)
-    include(${rapids-cmake-dir}/cpm/thrust.cmake)
-    rapids_cpm_thrust(NAMESPACE cuco
-                      BUILD_EXPORT_SET cuco-exports)
+# Use CPM to find or clone CCCL
+function(find_and_configure_cccl)
+    include(${rapids-cmake-dir}/cpm/cccl.cmake)
+    rapids_cpm_cccl(BUILD_EXPORT_SET cuco-exports)
```

```suggestion
    rapids_cpm_cccl(INSTALL_EXPORT_SET cuco-exports BUILD_EXPORT_SET cuco-exports)
```
Needs to be in the install set as well since we have a public dependency on the libcudacxx component